src/helpers/model-factory.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+class ModelFactory {
+
+  constructor () {
+    this.models = null;
```

Comment:
Not a blocker but in some of the code I've been working on I've started naming variables with underscore for variables not meant for direct access, worth doing here? (Can be done in a seperate PR)